_posts/2019-12-25-boost-quant-perf.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Boost Quantization Inference Performance on ARM CPU
+author: Zhenhua WANG, Zhao WU, Chunlei SANG, Yihua JIN
+date: 2019-12-25
```

Comment:
Could just be AliOS TVM Team

Reply:
Emmm... The team is here...

Reply:
What I means like this blog: https://tvm.apache.org/2018/03/23/nmt-transformer-optimize.
```
Author
This is a guest blogpost contributed by Alibaba Group’s Machine Translation Platform team and PAI-Blade team
```
So the author could just be:
```
author: Alibaba AliOS TVM Team
```

Reply:
Yes, I have seen this, not sure which should be used. Do we have guide line for this @tqchen ?

Reply:
Both ways should work fine